src/views/GameView.vue

```diff
@@ -128,13 +128,14 @@
 							</div>
 						</template>
 					</v-card>
-					<div
-						id="scrap"
-						class="rounded"
-					>
-						<p>Scrap</p>
-						<p>({{ scrap.length }})</p>
-					</div>
+					<scrap-dialog :scrap="scrap">
+						<template #activator>
+							<v-card id="scrap">
+								<h2>Scrap</h2>
```

Comment:
Let's use an h3 instead for lower weight on the page (since the scrap isn't THAT important most of the time)